hokusai/production.yml

```diff
@@ -31,23 +31,29 @@ spec:
         - name: team-web
           env:
             - name: PORT
-              value: "3000"
+              value: "8080"
             - name: NODE_ENV
               value: production
+            - name: DATADOG_TRACE_AGENT_HOSTNAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: NODE_OPTIONS
+              value: "--max_old_space_size=512"
           envFrom:
             - configMapRef:
                 name: team-environment
           image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/team:production
           imagePullPolicy: Always
           ports:
             - name: team-http
-              containerPort: 3000
+              containerPort: 8080
           resources:
             requests:
-              cpu: 200m
-              memory: 256Mi
-            limits:
+              cpu: 100m
```

Comment:
really spiky cpu [usage](https://app.datadoghq.com/dashboard/2n5-6tr-ypp/kubernetes-cpumemory-usage-by-deployment?from_ts=1616003018168&live=false&to_ts=1616015960186&tpl_var_deployment=team-web&tpl_var_env=production&tpl_var_hpa=team-web). reduce cpu per pod (so less allocation when nothing's going on) and raise hpa max replica (so more pods when usage comes).